### DISCUSSION TITLE
How to use Mongodb Collation in prisma

### DISCUSSION THREAD

**Discussion by divyam234** [prisma/prisma#18324] [Q&A] [answered]
Does prisma supports passing collation document without using  findRaw method.Also how to map raw query result to actual prisma model for mongodb as  findRaw method returns JsonObject.

**Comment by ludralph** [comment, marked as answer]
Hi @divyam234 👋 

Prisma does not support passing collation documents at the moment. However, there is a feature request to support this, as seen in this [issue](https://github.com/prisma/prisma/issues/13292). 

Also, from the documentation, the return type for the findRaw method is JsonObject, and there is no known method to map it to the Prisma model. I think you might want to open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) and detail your use-case.
